sig-service-catalog/charter.md

```diff
@@ -0,0 +1,128 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI).
+It allows application developers the ability to provision and consume cloud services natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
+- Provide a UX for Kubernetes users that is consistent with both the OSB API specification and traditional Kubernetes user interactions.
+- Align with the OSBAPI specification as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
+
+### Out of scope
+
+The following, non-exhaustive, items are out of scope:
+- Operation of OSBAPI Service Brokers.
+
+## Roles
+
+- Maintainers
+  - Responsible for approving, and reviewing, pull requests.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers are nominated by a chair and require unanimous consent by all chairs.
+  - Maintainers can be “retired” at the suggestion of a chair, and approved unanimously by the other chairs.
+
+- Chairs
+  - All maintainer’s roles.
+  - Responsible for project administration activities within the SIG and are non-technical
+    in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes leadership teams.
+  - Chairs MAY decide to step down at anytime and MAY propose a replacement, ideally from existing
+    maintainers. Use lazy consensus amongst chairs with fallback on majority vote to accept proposal.
```

Comment:
This is inconsistent with line 28 where is unanimous for new maintainers. Not sure why this is a lower bar. Let's keep it at unanimous since that what it is today.

Reply:
Thinking more about this - I think we should just separate this into two parts - steeping down and adding a new one. Stepping down can be at any time w/o a vote - we can't force someone to stay.  Then the replacement side is just following whatever rule we define for adding a new one.

Reply:
Agreed it doesn't need to be two separate rules. I'll follow up in a sec to make that consistent.